src/app/components/ChooseSeats.tsx

```diff
@@ -0,0 +1,98 @@
+import { FC } from 'react';
+import { Seat } from '../(site)/tickets/page';
+
+type Props = {
+  seatsArray: [] | { status: Seat; seatNumber: number }[];
+  ticketCount: number;
+  onSetSeatsArray: (
+    newArray: {
+      status: Seat;
+      seatNumber: number;
+    }[]
+  ) => void;
+};
+
+const ChooseSeats: FC<Props> = ({
+  seatsArray,
+  ticketCount,
+  onSetSeatsArray,
+}) => {
+  return (
+    <>
+      <h2 className="header-two">Välj platser</h2>
+
+      <div className="movie-screen-seat-container">
+        <div className="movie-screen-container">
+          <div className="movie-screen-left"></div>
+          <div className="movie-screen-middle"></div>
+          <div className="movie-screen-right"></div>
+        </div>
+
+        <div className="seat-container">
+          {seatsArray.map((seat) => {
+            return (
+              <div
+                key={seat.seatNumber}
+                onClick={() => {
```

Comment:
I found this part quite difficult to read, it works well but I think you can improve this part by giving your variables more detailed names.

Reply:
ok